opm/input/eclipse/Deck/DeckKeyword.hpp

```diff
@@ -66,6 +66,11 @@ namespace Opm {
         bool isDataKeyword() const;
         bool isDoubleRecordKeyword() const;
 
+        std::vector<int>& getIntData();
+        std::vector<double>& getRawDoubleData();
+
+
```

Comment:
excessive whitespace

Reply:
> excessive whitespace

Good catch.  I've trimmed these in the latest update.